frappe/translate.py

```diff
@@ -559,23 +559,28 @@ def read_csv_file(path):
 			newdata = [[ val for val in row ] for row in data]
 	return newdata
 
+
 def write_csv_file(path, app_messages, lang_dict):
 	"""Write translation CSV file.
 
 	:param path: File path, usually `[app]/translations`.
 	:param app_messages: Translatable strings for this app.
 	:param lang_dict: Full translated dict.
 	"""
+	if version_info[0] > 2:
+		write_mode = "w"
+
 	app_messages.sort(key = lambda x: x[1])
 	from csv import writer
-	with open(path, 'wb') as msgfile:
+	with open(path, write_mode) as msgfile:
 		w = writer(msgfile, lineterminator='\n')
 		for p, m in app_messages:
 			t = lang_dict.get(m, '')
 			# strip whitespaces
 			t = re.sub('{\s?([0-9]+)\s?}', "{\g<1>}", t)
 			w.writerow([p.encode('utf-8') if p else '', m.encode('utf-8'), t.encode('utf-8')])
```

Comment:
I have also modified this line as `w.writerow([p if p else '', m, t])` otherwise the content of the translated file is like
`b'DocType: My Doctype',b'My Doctype',b''`
instead of
`DocType: My Doctype,My Doctype,`

Reply:
Thanks for the edit